persistence/test/ibc_test.go

```diff
@@ -120,3 +121,180 @@ func TestGetIBCStoreEntry(t *testing.T) {
 		})
 	}
 }
+
+type attribute struct {
+	key   []byte
+	value []byte
+}
+
+func TestIBCSetEvent(t *testing.T) {
+	db := NewTestPostgresContext(t, 1)
```

Comment:
Leave a comment explaining how you're preparing the database for the rest of the tests.